src/workerd/server/json-logger.h

```diff
@@ -28,4 +29,31 @@ class JsonLogger: public kj::ExceptionCallback {
   bool loggingInProgress = false;
 };
 
+class StructuredLoggingProcessContext final: public kj::ProcessContext {
+  // A ProcessContext implementation that supports both plain text and structured JSON logging.
+  // This context wraps TopLevelProcessContext and adds the ability to emit log messages in
+  // JSON format when structured logging is enabled.
+
+ public:
+  explicit StructuredLoggingProcessContext(kj::StringPtr programName);
+
+  // Enable structured JSON logging. This can only be called once and cannot be reversed.
+  // When enabled: Log messages are formatted as JSON and sent to stdout
+  // When disabled: Log messages are sent as plain text to stderr (like TopLevelProcessContext)
```

Comment:
Are these comments accurate? because when structured logs are enabled the log messages can make it into stdout but also stderr now, no?

Also when structured logs are disabled log messages are not all sent to stderr (but to both stderr and stdout)

Or am I missing something?

Reply:
This should be stderr, not sure how or when it changed, it was stderr in previous commits.
Amended the comment to be a bit clearer.